lib/src/geometry/line_segment.dart

```diff
@@ -0,0 +1,104 @@
+import 'line.dart';
+import '../../extensions.dart';
+
+class LineSegment {
+  final Vector2 from;
+  final Vector2 to;
+
+  LineSegment(this.from, this.to);
+
+  /// If the line segment is outside of the hitbox
+  bool isOutside(List<Vector2> hitbox) {
```

Comment:
this might be more useful as a Rect or Poly? not clear what List<Vector2> hitbox is allowed be

Reply:
Removed this method completely, it's not used anymore.
The naming was a bit unclear too, if we need something similar in the future it should be very easy to add it.